viamonvif/xsd/onvif/body_response.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
TODO(seanp): Write tests against example body payload

Add to artifact?

Reply:
@nicksanford Do you think it is worth adding an artifact system to the viamrtsp repo at this point?

Reply:
I think this is sufficient for now.